static_files/js/map.js

```diff
@@ -1870,8 +1870,17 @@ function trackClick(e) {
   _gaq.push(['_trackEvent', category, action, label]);
 }
 
-function gaTracking () {
-  $('.ga-tracking').live('mousedown', trackClick);
+function trackShuttleClick(e) {
+  var $target = $(e.target),
+      category = 'Shuttle Info',
+      action = 'click',
+      label = $target.text();
+  _gaq.push(['_trackEvent', category, action, label]);
+}
+
+function gaTracking() {
+	$('.ga-tracking').live('mousedown', trackClick);
```

Comment:
Using `.delegate()`, particularly for the old version of jquery that Map is running on, is preferred over `.live()`:  http://api.jquery.com/live/#entry-longdesc